tfx/examples/cifar10/cifar10_pipeline_native_keras.py

```diff
@@ -0,0 +1,218 @@
+# trained model here.
+_serving_model_dir_lite = os.path.join(_cifar10_root, 'serving_model_lite',
+                                       _pipeline_name)
+
+# Directory and data locations.  This example assumes all of the images,
+# example code, and metadata library is relative to $HOME, but you can store
+# these files anywhere on your local filesystem.
+_tfx_root = os.path.join(os.environ['HOME'], 'tfx')
+_pipeline_root = os.path.join(_tfx_root, 'pipelines', _pipeline_name)
+# Sqlite ML-metadata db path.
+_metadata_path = os.path.join(_tfx_root, 'metadata', _pipeline_name,
+                              'metadata.db')
+
+# Pipeline arguments for Beam powered Components.
+_beam_pipeline_args = [
+    '--direct_running_mode=multi_processing',
+    # 0 means auto-detect based on on the number of CPUs available
+    # during execution time.
+    '--direct_num_workers=0',
+]
+
+def _create_pipeline(pipeline_name: Text, pipeline_root: Text, data_root: Text,
+                     module_file: Text, serving_model_dir_lite: Text,
+                     metadata_path: Text,
+                     beam_pipeline_args: List[Text]) -> pipeline.Pipeline:
+  """Implements the CIFAR10 image classification pipeline using TFX."""
+  # This is needed for datasets with pre-defined splits
+  # Change the pattern argument to train_whole/* and test_whole/* to train
+  # on the whole CIFAR-10 dataset
+  input_config = example_gen_pb2.Input(splits=[
+      example_gen_pb2.Input.Split(name='train', pattern='train/*'),
+      example_gen_pb2.Input.Split(name='eval', pattern='test/*')])
+
+  examples = external_input(data_root)
+
+  # Brings data into the pipeline.
+  example_gen = ImportExampleGen(input=examples, input_config=input_config)
+
+  # Computes statistics over data for visualization and example validation.
+  statistics_gen = StatisticsGen(examples=example_gen.outputs['examples'])
+
+  # Generates schema based on statistics files.
+  schema_gen = SchemaGen(
+      statistics=statistics_gen.outputs['statistics'], infer_feature_shape=True)
+
+  # Performs anomaly detection based on statistics and data schema.
+  example_validator = ExampleValidator(
+      statistics=statistics_gen.outputs['statistics'],
+      schema=schema_gen.outputs['schema'])
+
+  # Performs transformations and feature engineering in training and serving.
+  transform = Transform(
+      examples=example_gen.outputs['examples'],
+      schema=schema_gen.outputs['schema'],
+      module_file=module_file)
+
+  # Uses user-provided Python function that trains a model.
+  # When traning on the whole dataset, use 18744 for train steps, 156 for eval steps.
+  # 18744 train steps correspond to 24 epochs on the whole train set, and 156 eval
+  # steps correspond to 1 epoch on the whole test set. The configuration below is for
+  # training on the dataset we provided in the data folder, which has 100 train and
+  # 100 test samples. The 160 train steps correspond to 40 epochs on this tiny train set,
```

Comment:
Lint format? over 80 per line?

Reply:
hmmm I have linted the file and it was within 80 characters.